cli_tools/gce_vm_image_import/main.go

```diff
@@ -18,119 +18,117 @@ package main
-	sourceFile           = flag.String("source_file", "", "Google Cloud Storage URI of the virtual disk file	to import. For example: gs://my-bucket/my-image.vmdk")
-	sourceImage          = flag.String("source_image", "", "Compute Engine image from which to import")
-	noGuestEnvironment   = flag.Bool("no_guest_environment", false, "Google Guest Environment will not be installed on the image.")
-	family               = flag.String("family", "", "Family to set for the translated image")
-	description          = flag.String("description", "", "Description to set for the translated image")
-	network              = flag.String("network", "", "Name of the network in your project to use for the image import. The network must have access to Google Cloud Storage. If not specified, the network named default is used.")
-	subnet               = flag.String("subnet", "", "Name of the subnetwork in your project to use for the image import. If	the network resource is in legacy mode, do not provide this property. If the network is in auto subnet mode, providing the subnetwork is optional. If the network is in custom subnet mode, then this field should be specified. Zone should be specified if this field is specified.")
-	zone                 = flag.String("zone", "", "Zone of the image to import. The zone in which to do the work of importing the image. Overrides the default compute/zone property value for this command invocation.")
-	timeout              = flag.String("timeout", "", "Maximum time a build can last before it is failed as TIMEOUT. For example, specifying 2h will fail the process after 2 hours. See $ gcloud topic datetimes for information on duration formats.")
-	project              = flag.String("project", "", "project to run in, overrides what is set in workflow")
-	scratchBucketGcsPath = flag.String("scratch_bucket_gcs_path", "", "GCS scratch bucket to use, overrides what is set in workflow")
-	oauth                = flag.String("oauth", "", "path to oauth json file, overrides what is set in workflow")
-	ce                   = flag.String("compute_endpoint_override", "", "API endpoint to override default")
-	gcsLogsDisabled      = flag.Bool("disable_gcs_logging", false, "do not stream logs to GCS")
-	cloudLogsDisabled    = flag.Bool("disable_cloud_logging", false, "do not stream logs to Cloud Logging")
-	stdoutLogsDisabled   = flag.Bool("disable_stdout_logging", false, "do not display individual workflow logs on stdout")
-	kmsKey               = flag.String("kms_key", "", "ID of the key or fully qualified identifier for the key. This flag must be specified if any of the other arguments below are specified.")
-	kmsKeyring           = flag.String("kms_keyring", "", "The KMS keyring of the key.")
-	kmsLocation          = flag.String("kms_location", "", "The Cloud location for the key.")
-	kmsProject           = flag.String("kms_project", "", "The Cloud project for the key")
```

Comment:
I don't see these in the args package. Please don't remove any flags, even if they are not doing anything, as they are now part of the "API" (and they will likely be used once we can implement their use). It's ok you removed piping these flags through to the importer, as that's internal to our code.

Reply:
done